snap/snapcraft.yaml

```diff
@@ -0,0 +1,70 @@
+name: zed-editor
+title: Zed
+base: core24
+version: '0.149.5'
```

Comment:
This should `adopt-info: zed` and the version should be set in one of the override steps with `craftctl set version=...`

Reply:
Their appstream metainfo file will again need to be staged into the snap file. And that too is also filled with placeholders.